.husky/commit-msg

```diff
@@ -1 +1,5 @@
-npx --no-install commitlint --edit "$1"
+
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit "$1"
```

Comment:
GitHub complains if you don't have an empty line at the end. Configure your editor with editorconfig or something similar to add it automatically when you save.

Reply:
Done, configured my editor accordingly